src/Documentation/grains/grain_identity.md

```diff
@@ -0,0 +1,123 @@
+---
+layout: page
+title: Grain Identity
+---
+
+# Grain Identity
+
+In object-oriented environments, the identity of an object is hard to distinguish from a reference to it.
+Thus, when an object is created using new, the reference you get back represents all aspects of its identity except those that map the object to some external entity that it represents.
+
+In distributed systems, object references cannot represent instance identity, since references are typically limited to a single address space.
+That is certainly the case for .NET references.
+Furthermore, a virtual actor must have an identity regardless of whether it is active, so that we can activate it on demand.
```

Comment:
IMO we should write "grain" instead of "virtual actor" here (and replace actor with grain on line 18 below, too). @sergeybykov, what do you think?

Reply:
I agree - we should be talking about grains, and not confuse people by using actors or virtual actors interchangeably.

Reply:
Fixed - thank you!